ts/Dashboards/Plugins/HighchartsComponent.ts

```diff
@@ -656,9 +659,22 @@ class HighchartsComponent extends Component {
                     }
                 }
 
+                // Disable dragging on series, which were created out of a
+                // columns which are created by MathModifier.
+                const shouldBeDraggable = !(
+                    modifierOptions?.type == 'Math' &&
+                    (modifierOptions as MathModifierOptions)
+                    .columnFormulas?.some(
+                        (formula) => formula.column == seriesName
+                    )
+                );
+
                 return chart.addSeries({
                     name: seriesName,
-                    id: `${storeTableID}-series-${index}`
+                    id: `${storeTableID}-series-${index}`,
+                    dragDrop: {
+                        draggableY: shouldBeDraggable
```

Comment:
We need this info in API - any thoughts on where we should place it? Not in Highcharts API obviously.

Perhaps in the general description of the Chart component? We could list default options for the series (and the chart) there.